### ISSUE TITLE
Any working example on codepen please?

### ISSUE THREAD

**Issue by agenciaRSE** [BrookeDot/SimplerWeather#6]
I have try to add it to a website was using simpleweather but was not working before and i can't make it work.

Any working example on codepen please? 

Would be awesome.

**Comment by aspirinnz** [comment]
I don't think that a public example of going to be available because the API calls now have a cost involved and seems like the API key needs to be saved in a proxy php file so would require an external file be referenced. But, yes, I do wish there were some more complete example files.
